app/src/internal/java/com/kickstarter/ui/adapters/FeatureFlagsAdapter.kt

```diff
@@ -4,22 +4,24 @@ import android.view.LayoutInflater
 import android.view.ViewGroup
 import com.kickstarter.R
 import com.kickstarter.databinding.EmptyViewBinding
+import com.kickstarter.databinding.ItemFeatureFlagBinding
+import com.kickstarter.model.FeatureFlagsModel
 import com.kickstarter.ui.viewholders.EmptyViewHolder
 import com.kickstarter.ui.viewholders.FeatureFlagViewHolder
 import com.kickstarter.ui.viewholders.KSViewHolder
 
-class FeatureFlagsAdapter : KSAdapter() {
+class FeatureFlagsAdapter(private val delegate: FeatureFlagViewHolder.Delegate) : KSAdapter() {
```

Comment:
@Arkariang Do we normally use the namespace to identify an interface like this or is the standard `FeatureFlagViewHolderDelegate` instead?

Reply:
On android for this cases should probably be something like `FeatureFlagViewHolder.ViewListener`, but throughout  the app there is mostly called `Delegate` - the android app was starter by some iOS folks - so I see where it comes from see this one as example [ProjectCardViewHolder](https://github.com/kickstarter/android-oss/blob/fa7a573b7f5e0879ca903f394b99660ab85a7383/app/src/main/java/com/kickstarter/ui/viewholders/ProjectCardViewHolder.kt#L35) .
But yeah I agree, a naming more android-related would be nice